app/admin.py

```diff
@@ -108,6 +109,8 @@ def post(self):
                         'deactivation_message_html',
                         'keywords',
                         'launch_status',
+                        'admin_user_email',
+                        'thres_unreviewed_notes',
```

Comment:
Can you use clearer name here e.g., "unreviewed_notes_email_threshold"? In general, we should avoid abbreviations e.g., "thres" unless it's widely used (either in the code base or in the world).